_data/tables/transform-high-performing-org.yml

```diff
@@ -188,7 +251,7 @@
       data:
         list: false
         fr: 
-        en: Create system to store and analyze key metrics from teams
+        en: Analyze progress and key metrics from teams to update this stratgegy quarterly
```

Comment:
Analyze progress and key metrics from teams to update this stratgegy quarterly
==>
Reassess metrics and their respective impacts on a bi-annual basis

I mean, if metric y was meant to improve metric x, and y improved but x did not, when reassess if y is actually having the intended consequences. Change metric, or investigate why the increase in y did not have the intended impact

Reply:
This action is the the IT strategy team to look at progress and update this strategy.  Teams should be continuously reassessing as per the Guiding policy: "IITB teams seek to understand the current condition, set measurable targets for a future date and experiment with ways to achieve these targets until the future date is reached.
When the results have been captured and new targets are set, repeat the process."

Reply:
Aye, the intent was to make the actions more like actions and less hand wavey. Analyzing on its own isnt a useful action, its the outcome of said analysis. The outcome of this analysis should be an adjustment of key metrics. So the action should be the action instead of the thing we need to do before the action